Remotion/Web/Core/UI/HtmlHeadAppenderExtensions.cs

```diff
@@ -31,13 +31,24 @@ public static class HtmlHeadAppenderExtensions
     /// <summary>
     /// Registers Utilities.js in non-themed HTML folder of Remotion.Web.dll.
     /// </summary>
+    public static void RegisterWebClientScript (this HtmlHeadAppender htmlHeadAppender)
```

Comment:
```suggestion
    public static void RegisterWebClientScriptInclude (this HtmlHeadAppender htmlHeadAppender)
```

Naming: "ScriptInclude" is also ment to represent the html DOM element.